tests/covscripts/coverage.py

```diff
@@ -71,20 +71,20 @@ def develop(root_dir, coverage_file_name_txt, coverage_file_name_json):
 def branch(root_dir, coverage_file_name_txt, coverage_file_name_json):
   """run and compare coverage for the non-develop, peripheral branches"""
   file_path_curcov_json = os.path.join(root_dir, coverage_file_name_json)
-  curcov_percentage_ftdt = curcov(file_path_curcov_json)
+  cur_cov = curcov(file_path_curcov_json)
+  prev_cov = None
 
   with open(coverage_file_name_txt, 'r') as f:
     prevcov_file = f.readline().strip()
-    prevcov_percentage_ftdt = float(prevcov_file)
+    prev_cov = float(prevcov_file)
 
-  print(
-      f"Current Testing Coverage for current branch is: {curcov_percentage_ftdt}%"
-  )
-  print(
-      f"Current Testing Coverage for develop branch is: {prevcov_percentage_ftdt}%"
-  )
-  if curcov_percentage_ftdt < prevcov_percentage_ftdt:
-    raise ValueError('Code Coverage Decreased')
+  print(f"Current Testing Coverage for current branch is: {cur_cov}%")
+  print(f"Current Testing Coverage for develop branch is: {prev_cov}%")
+  prct_change = (prev_cov - cur_cov) / prev_cov * 100
+  if prct_change > 1:
```

Comment:
How about we reduce this tolerance to maybe 0.1 or 0.2 percent

Reply:
i would not go below 0.5. Mine fails with something like 0.25 so it would not be enough